src/tokens/transformation/android/formats/fontStyle.js

```diff
@@ -1,5 +1,5 @@
 const { fileHeader } = require('style-dictionary').formatHelpers
-const camelCase = require('../common/camelCaseHelper')
+const changeCase = require('change-case')
```

Comment:
Hey, we still don't have a dependency on `change-case`. We should either:
1. Implement our own version of this (my preference)
2. Add the library to `package.json`

Reply:
change-case  is included as a part of style-dictionary  : https://github.com/brave/leo/blob/4fd60db98b946355e38f781586502cd26d069b34/package-lock.json#L33326

(https://bravesoftware.slack.com/archives/D06HLN2SDJP/p1710861466483659)
https://github.com/brave/leo/blob/4fd60db98b946355e38f781586502cd26d069b34/package-lock.json#L8218

Reply:
cc'ing @petemill - should we depend on this directly? Not for this PR, just in general